src/main/java/network/brightspots/rcv/Tabulator.java

```diff
@@ -917,7 +915,6 @@ private void recordSelectionForCastVoteRecord(
             continue;
           }
 
-          // TODO: it's weird to have this check here
```

Comment:
It seems like the rest of the stuff below here may have been part of the TODO? Does it make sense on its own?
```
// handle testing in unit tests, remove this assert, and add break statement
// at end of this for loop
```

Reply:
Oops. I just went ahead and did what @moldover suggested, except for adding a unit test. I'll create a new issue for that part.